at.colpanic.e4raptycho.target/at.colpanic.e4raptycho.target.target

```diff
@@ -4,16 +4,73 @@
 <target name="RAP" sequenceNumber="1583699636">
   <locations>
     <location includeMode="slicer" includeAllPlatforms="true" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.rap.e4.sdk.feature.feature.group" version="3.11.0.20191127-1205"/>
-      <unit id="org.eclipse.rap.e4.equinox.target.feature.feature.group" version="3.11.0.20191211-1200"/>
-      <unit id="org.eclipse.rap.e4.feature.feature.group" version="3.11.0.20191127-1205"/>
-      <repository location="http://download.eclipse.org/rt/rap/3.11/e4"/>
+		<unit id="org.eclipse.rap.feature.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.rap.sdk.feature.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.rap.examples.feature.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.rap.equinox.target.feature.feature.group" version="0.0.0"/>
+		<unit id="com.ibm.icu" version="0.0.0"/>
+        <unit id="javax.servlet" version="0.0.0"/>
```

Comment:
@col-panic This may be simplified but as maven guy the management of plugins is still a bit mysterious for me